docs/guide/search/elastic.md

```diff
@@ -0,0 +1,573 @@
+    - `path` - By default, path is not used.
+    The default value is `null`.
+    If you have several Elasticsearch instances that run on the same host, and want to differentiate
+    them, you can define a path for each instance.  
+    - `user`/`pass` - Credentials, if needed to login to the host.
+    Default values are `null`.
+
+    You list the addresses of cluster nodes under the `hosts` key:
+
+    ``` yaml
+    ez_platform_elastic_search_engine:
+      connections:
+        <connection_name>:
+          hosts:
+            - '127.0.0.1:9200'
+            # - ...
+    ```
+
+    There are several ways that you can use to pass host parameters.
+    The easiest one is to pass them as a string:
+
+    ``` yaml
+    - https://<my.elasticsearch.domain>:9200/<path>/
+    ```
+
+    You can also pass the host configuration as an object that lists parameter-value pairs,
+    for example, when your authentication settings contain special characters.  
+
+    ``` yaml
+    - { host: '<my.elasticsearch.domain>', scheme: 'http', port: 9200, path: '/', user: <username>, pass: <password> }
+    ```
+
+    Your cluster connection configurations should be similar to the following examples:
+
+    ``` yaml
+    ezplatform_elastic_search_engine:
+      connections:
+    	  simple:
+    	    hosts:
+    	      - '127.0.0.1:9200'
+    	      - '127.0.0.1:9201'
+    	      - '127.0.0.1:9202'
+
+    	    localhost:
+    	      debug: true
+    	      hosts:
+    	        - "127.0.0.1:9200"
+    	        - "b.elasticsearch.loc:9200"
+    	        - "c.elasticsearch.loc:9200"
+
+    	    intranet:
+    	      debug: true
+    	      hosts:
+    	        - "c.elasticsearch.loc:9200"
+
+    	  default_connection: simple
+    ```
+
+    ###### Configure the multi-node cluster behaviour
+
+    When you configure a cluster-based connection, and the cluster consists of many nodes, you can
+    choose strategies that govern how the cluster reacts to changing operating conditions, or how
+    workload is distributed among the nodes.
+
+    Connection pool
+
+    With this setting you decide how a list of hosts that form a cluster is managed.
+    The list of active hosts tends to change in time, due to different reasons, such as connectivity
+    issues, host malfunction, or the fact that you add new hosts to the cluster to increase
+    its performance.
+    By default, the `StaticNoPingConnectionPool` setting is used.
+
+    You can change the default setting with the following key:
+
+    ``` yaml
+    <connection_name>:
+      # ...
+      connection_pool: Elasticsearch\ConnectionPool\<connection_pool_name>
+    ```
+
+    For more information and a list of available choices, see [Connection pool](https://www.elastic.co/guide/en/elasticsearch/client/php-api/7.x/connection_pool.html).
+
+    One of possible choices is the `sniffingConnectionPool` setting, which provides increased flexibility.
```

Comment:
Add why you select this setting out of the others. Is it particularly recommended for Platform? If not, maybe not worth mentioning